src/Extractor/BaseExtractor.php

```diff
@@ -63,7 +64,7 @@ public function __construct(array $parameters, array $state, LoggerInterface $lo
 
         try {
             $proxy->call(function (): void {
-                $this->db = $this->createConnection($this->dbParameters);
+                $this->db = $this->createConnection($this->createDatabaseConfig($this->dbParameters));
```

Comment:
Nebolo by to dobre v `__construct` ulozit tuto hodnotu aj do `protected $dbConfig` ? aby sa to dalo v pripade potreby pouzit v triedach, ktore od tejto dedia + ... nemusel by si v kode 2x volat `createDatabaseConfig`

Reply:
jj vyměnil jsem to za dpParams který by tam už neměly být potřeba